sh/tests/head-and-tail_test.sh

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env bash
+
+# Unofficial Bash Strict Mode
+set -euo pipefail
+IFS='
+'
+
+echo insecure >> ~/.curlrc
+caddy start &>/dev/null
+
```

Comment:
I would check if the file has been submitted correctly as it has been done in the exercise `to-git-or-not-to-git` (https://github.com/01-edu/public/blob/master/sh/tests/to-git-or-not-to-git_test.sh)

Reply:
fixed